fdbclient/vexillographer/fdb.options

```diff
@@ -125,6 +125,9 @@ description is not currently required but encouraged.
     <Option name="client_buggify_section_fired_probability" code="83"
             paramType="Int" paramDescription="probability expressed as a percentage between 0 and 100"
             description="Set the probability of an active CLIENT_BUGGIFY section being fired. A section will only fire if it was activated" />
+    <Option name="distributed_client_tracer" code="90"
+            paramType="String" paramDescription="Distributed tracer type. Choose from none, log_file, network_async, or network_lossy"
+            description="Set a tracer to run on the client. Should be set to the same value as the tracer set on the server." />
```

Comment:
What happens if they are not set to same value?

Reply:
Tracing will be incomplete on both client and server. Because fdbclient is where transactions originate, the read version is fetched, etc..., if the client sets the tracer to `none` but the server sets it to `log_file`, the log files will contain traces from events such as proxy batch commits and storage server updates, but won't contain info about the transaction being created or other fdbclient events. Markus has proposed a global configuration framework to solve this problem, so ideally this option will be removed at some point in the future.